MANIFEST

```diff
@@ -346,8 +348,13 @@ lib/LaTeXML/Engine/TeX_Tables.pool.ltxml
 lib/LaTeXML/Engine/plain.pool.ltxml
 lib/LaTeXML/Engine/LaTeX.pool.ltxml
 lib/LaTeXML/Engine/eTeX.pool.ltxml
-lib/LaTeXML/Engine/expl3.pool.ltxml
 lib/LaTeXML/Engine/pdfTeX.pool.ltxml
+lib/LaTeXML/Engine/plain_bootstrap.pool.ltxml
+lib/LaTeXML/Engine/plain_constructs.pool.ltxml
+lib/LaTeXML/Engine/latex.pool.ltxml
```

Comment:
This breaks case-insensitive filesystems! You can't have LaTeX and latex. That's probably why Windows fails. I am surprised you didn't have problems on macOS.

Reply:
Ouch! That's probably the problem, indeed. And it was such a nice naming convention...

Reply:
Woohoo! That was the problem! Where would we be without your? Thanks!!

Reply:
I can be more annoying! Would you be able to rebase the latex/LaTeX difference away? It breaks e.g. `git bisect` because you can't even checkout the repository correctly. It is not a big deal though, at all.

Reply:
Does this make it hard to test as a PR? (other than `/tmp`)  The expectation is that the PR would eventually be squashed when merging, so the name clash should disappear in the long run.

Reply:
> Does this make it hard to test as a PR?

Yes, this was the first time even I tried `git bisect` and it stopped working after two steps, claiming that the worktree was modified (because one of latexml.pool and LaTeXML.pool always had the wrong content).

I suppose if you squash merge, then the problem will indeed go away on its own.